Lib/annotationlib.py

```diff
@@ -6,7 +6,14 @@
 import sys
 import types
 
-__all__ = ["Format", "ForwardRef", "call_annotate_function", "get_annotations"]
+__all__ = [
+    "Format",
+    "ForwardRef",
+    "call_annotate_function",
+    "call_evaluate_function",
+    "get_annotate_function",
```

Comment:
These two were missing. The rest of the changes in this file are just formatting.